{{cookiecutter.buildfolder}}/pyproject.toml

```diff
@@ -1,17 +1,20 @@
 [tool.poetry]
-name = "data-model-examples"
+name = "cognite-toolkit"
```

Comment:
A bit general name maybe

Reply:
Had a discussion with tech writers on this. TBD on how to present this, so I go for something that doesn't break other naming. I hope it can go away soon and we only need some tools to load example data.